test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -53,10 +54,14 @@ def test_tensorflow_train_mnist_cpu(tensorflow_training, ec2_connection, cpu_onl
 @pytest.mark.integration("horovod")
 @pytest.mark.model("resnet")
 @pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
-def test_tensorflow_with_horovod_gpu(tensorflow_training, ec2_connection, gpu_only):
+def test_tensorflow_with_horovod_gpu(tensorflow_training, ec2_instance_type, ec2_connection, gpu_only):
     test_script = TF1_HVD_CMD if is_tf1(tensorflow_training) else TF2_HVD_CMD
-    execute_ec2_training_test(ec2_connection, tensorflow_training, test_script,
-                              large_shm=True if "p2.8xlarge" in TF_EC2_GPU_INSTANCE_TYPE else False)
+    execute_ec2_training_test(
+        connection=ec2_connection,
+        ecr_uri=tensorflow_training,
+        test_cmd=test_script,
+        large_shm=bool(re.match(r"(p2\.8xlarge)|(g3\.16xlarge)", ec2_instance_type))
```

Comment:
Did you try this on different gpu instance types this test might run on? If so, can you update the tests run section?

Reply:
Currently experimenting with g3.4xlarge. Will update as soon as those tests run.